.github/workflows/release-check.yml

```diff
@@ -14,6 +14,8 @@ env:
   RUST_BACKTRACE: 1
   SHELL: /bin/bash
   CARGO_INCREMENTAL: 0
+  # Force attestation verification and fail build if verification fails.
+  MOZJS_ATTESTATION: forceforce
```

Comment:
typo?

Reply:
No. The name is ugly and subject to bikeshedding. Below a copy of the comment in the code explaining the available values.

>  ```
>      // MOZJS_ATTESTATION values:
>      // unset -> no verification
>      // force -> force verification and fail if `gh` is not installed or recent enough
>      // set to any other value -> enable verification if `gh` installed and new enough.
>      // Note: If verification is enabled and fails, then we fall back to compiling from source.
>      // forceforce -> As a debug option we provide forceforce, which fails the build instead
>      //               of falling back to compiling from source if verification fails

The `force` option exists to force verification of the artifact. Since we can't assume `gh` is installed (or new enough, e.g. fedora packages a quite outdated version), the default is to just accept the downloaded artifact, if `gh` is not available.
When artifact verification is possible AND fails, then I think it makes sense to fallback to compiling from source instead of failing the build. 
For debugging purposes, we do actually want the build to fail, otherwise it's hard to tell if the artififact verification is working as expected (hence the more strict `forceforce` option, which I would consider an internal debug option).

Reply:
maybe we can use unset/lenient/strict as the possible values.

Reply:
Do you also have a suggestion for the fourth option? ExtraStrict?

Reply:
Do we need four modes? 
unset -> no verification if not possible
lenient -> verify if possible, but fallback to source if not possible or fails verification
strict -> always verify

Reply:
>  unset -> no verification if not possible

I would be fine with that. In that case we could add an explicit  `off` switch, to disable verification, even if possible.